pennylane/workflow/qnode.py

```diff
@@ -273,9 +273,10 @@ class QNode:
               rule for all supported quantum operation arguments, with finite-difference
               as a fallback.
 
-            * ``"hadamard"``: Use the analytic hadamard gradient test
-              rule for all supported quantum operation arguments. More info is in the documentation
-              :func:`qml.gradients.hadamard_grad <.gradients.hadamard_grad>`.
+            * ``"hadamard"``: Use the standard analytic hadamard gradient test rule for
+             all supported quantum operation arguments. More info is in the documentation
+             for :func:`qml.gradients.hadamard_grad <.gradients.hadamard_grad>`. Reversed,
+             direct, and reversed-direct modes can be selected via a ``"mode"`` in ``gradient_kwargs``.

```

Comment:
```suggestion
            * ``"hadamard"``: Use the standard analytic hadamard gradient test rule for
              all supported quantum operation arguments. More info is in the documentation
              for :func:`qml.gradients.hadamard_grad <.gradients.hadamard_grad>`. Reversed,
              direct, and reversed-direct modes can be selected via a ``"mode"`` in ``gradient_kwargs``.
```
Hope this can comfort Sphinx.